package.json

```diff
@@ -7,8 +7,12 @@
     "test": "nyc grunt test",
     "fix-lint-errors": "eslint --fix tasks/ lib/ test/*.js"
   },
+  "files": [
```

Comment:
Cool!  I didn't know about the package.json `files` block.  I definitely prefer it to `.npmignore`.

Reply:
It's preferable, isn't it? I see whitelisting less-error-prone to blacklisting too.